src/frontend/src/components/Banner/Banner.jsx

```diff
@@ -219,7 +219,7 @@ export default function Banner() {
       <Grid
         container
         spacing={0}
-        direction="row"
+        direction="column"
```

Comment:
Same comment as above, this undoes recent commit https://github.com/Seneca-CDOT/telescope/commit/d2b2e08ca8bd86ea58c1c7d195fb7b686e1bfa5d.